app/models/hydrus/processable.rb

```diff
@@ -17,24 +17,14 @@ def complete_workflow_step(step)
   # Calls the workflow service to mark that step as completed.
   def update_workflow_status(step, status)
     workflow_client.update_status(druid: pid,
-                                  workflow: Dor::Config.hydrus.app_workflow,
+                                  workflow: Settings.hydrus.app_workflow,
                                   process: step,
                                   status: status)
-    workflows_content_is_stale
```

Comment:
I removed this method call (and the method) since they assumed workflows was an editable datastream.